25_exceptions_lab/02.py

```diff
@@ -5,3 +5,16 @@
 Alert the user politely if there was any problem opening the file
 """
 
+import sys
+import msvcrt
+
+(_,file1) = sys.argv
+
+try :
+    num_lines = sum(1 for line in open(file1))
+except Exception as e:
```

Comment:
Better to also print the exception itself: It may contain valuable information about what went wrong